src/block/accordion/frontend-accordion.js

```diff
@@ -51,6 +55,22 @@ class StackableAccordion {
 						}, ANIM_OPTS )
 					}
 
+					// When the animation is done, remove text selection from double click
+					// and allow text selection again
+					el.anim.onfinish = el.anim.oncancel = () => {
+						if ( window.getSelection ) {
+							// eslint-disable-next-line @wordpress/no-global-get-selection
+							const selection = window.getSelection()
+							if ( selection.removeAllRanges ) {
+								selection.removeAllRanges()
+							}
+						} else if ( document.selection ) {
+							// For older IE browsers
+							document.selection.empty()
+						}
+						content.style.userSelect = 'auto'
```

Comment:
I see that you are clearing the selection because the `userSelect` above was not taking full effect and you can still highlight text when opening an accordion. Applying the style directly on the element fixes things, and now you do not need all of the selection removal code.

Also, to me, if the fix only partially works and you needed to add some selection removal code to compensate is an indicator that there should be a cleaner solution 👍

```suggestion
						el.style.userSelect = 'auto'
```